src/main/java/org/jabref/logic/util/StandardFileType.java

```diff
@@ -52,6 +52,11 @@
     }
 
     public static FileType newFileType(String... extensions) {
+        for (int i = 0; i < extensions.length; i++) {
```

Comment:
A normal for each loop should also work

Reply:
Got it.

Reply:
From what I've read online, you can't modify elements of an array in-place with a foreach loop, which is what I'm doing. See the line `extensions[i] = extensions[i].substring(extensions[i].indexOf('.') + 1);`. Am I mistaken?